google-datacatalog-connectors-commons/src/google/datacatalog_connectors/commons/datacatalog_facade.py

```diff
@@ -308,8 +308,9 @@ def upsert_tags(self, entry, tags):
             tag_to_create = tag
             tag_to_update = None
             for persisted_tag in persisted_tags:
+                # column name is case insensitive.
```

Comment:
```suggestion
                # The column field is not case sensitive.
```
Nitpick: I believe _not case sensitive_ reads better than _case insensitive_. 😃